Source/Core/DolphinWX/Debugger/MemoryWindow.h

```diff
@@ -45,7 +47,8 @@ class CMemoryWindow : public wxPanel
   void OnDumpFakeVMEM(wxCommandEvent& event);
   void OnMemCheckOptionChange(wxCommandEvent& event);
 
-  wxButton* btnSearch;
+  wxButton* btnFindNext;
```

Comment:
These should be normalized to the `m_` convention instead of proliferating the incorrect naming convention.